app/helpers/locomotive/current_site_metafields_helper.rb

```diff
@@ -74,7 +74,8 @@ def hint
       def type
         @type ||= case (type = @attributes['type'].try(:to_sym))
         when :boolean then :toggle
-        when :text    then :rte
+        when :text
+          @attributes['format'] == 'raw' ? :text : :rte
```

Comment:
Expected and desired behaviour regarding this feature would be `@attributes['format'] == 'html' ? :rte : :text` however this would likely break backwards compatibility